src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +76,20 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    private void Sorting() {
```

Comment:
Missing header comments, and naming does not follow coding standard. Names should be in camelCase.